packages/fnlfmt/package.yaml

```diff
@@ -0,0 +1,37 @@
+---
+name: fnlfmt
+description: Formatter for Fennel code
+homepage: https://git.sr.ht/~technomancy/fnlfmt
+licenses:
+  - MIT
+languages:
+  - Fennel
+categories:
+  - Formatter
+
+source:
+  # renovate:datasource=git-tags
+  id: pkg:generic/https://git.sr.ht/~technomancy/fnlfmt@0.3.2
+  build:
+    - target: unix
+      env:
+        URL: "https://git.sr.ht/~technomancy/fnlfmt/archive/{{version}}.tar.gz"
+      run: |
+        curl -L -o fnlfmt.tar.gz "$URL"
+        tar --no-same-owner -xvf fnlfmt.tar.gz
+        cd fnlfmt-*
+        echo "#!/usr/bin/env lua" > ../fnlfmt
+        LUA_PATH="./?.lua" ./fennel --require-as-include --compile cli.fnl >> ../fnlfmt
+        chmod +x ../fnlfmt
+      bin: fnlfmt
+    - target: win_x64
+      run: |
+        curl -L -o fnlfmt.tar.gz "https://git.sr.ht/~technomancy/fnlfmt/archive/0.3.2.tar.gz"
+        tar -xvf fnlfmt.tar.gz
+        cd fnlfmt-*
+        echo "@echo off" > ../fnlfmt.bat
+        echo "lua -e \"package.path='./?.lua;'..package.path\" fennel --require-as-include --compile cli.fnl | lua" >> ../fnlfmt.bat
```

Comment:
Just a sidenote, this would not work on Windows because it uses powershell syntax (network calls would preferably be done with `Invoke-WebRequest`). Not sure why tests are passing 🤔